extensions/machine-learning-services/package.json

```diff
@@ -27,28 +28,66 @@
   "contributes": {
     "commands": [
       {
-        "command": "ml.command.managePackages",
-        "title": "%ml.command.managePackages%"
+        "command": "mls.command.managePackages",
+        "title": "%mls.command.managePackages%"
+      },
+      {
+        "command": "mls.command.odbcdriver",
+        "title": "%mls.command.odbcdriver%"
+      },
+      {
+        "command": "mls.command.mlsdocs",
+        "title": "%mls.command.mlsdocs%"
+      }
+    ],
+    "dashboard.tabs": [
+      {
+        "id": "ml-server",
+        "description": "Machine Learning Services",
+        "provider": "MSSQL",
```

Comment:
nit: but you updated the loc, commands and other things to mls. - why not this one?